src/generators/app.ts

```diff
@@ -279,14 +279,15 @@ class App extends Generator {
     }
 
     if (['sfdx-plugin', 'plugin', 'multi'].includes(this.type)) {
-      this.pjson.scripts.prepack = nps.series('oclif-dev manifest', 'oclif-dev readme', 'npm shrinkwrap')
+      this.pjson.scripts.prepack = nps.series(`${rmrf} lib`, 'tsc -b', 'oclif-dev manifest', 'oclif-dev readme', 'npm shrinkwrap')
```

Comment:
I should probably put this behind the this.ts check, but the plugin generator only does tsc.